test/SlowTests/ExtensionPoints/ExtensionPointsTests.cs

```diff
@@ -336,7 +337,7 @@ exit 1
             X509Certificate2 serverCertificate;
             try
             {
-                serverCertificate = new X509Certificate2(certificates.ServerCertificatePath, (string)null, X509KeyStorageFlags.Exportable | X509KeyStorageFlags.MachineKeySet);
+                serverCertificate = new X509Certificate2(certificates.ServerCertificatePath, (string)null, CertificateLoaderUtil.FlagsForExport | X509KeyStorageFlags.MachineKeySet);
```

Comment:
shouldn't we use the LoadUtil here?

Reply:
As I see it, this is a test so there is no need to change it.

Reply:
but it generates extra files in windows machine key sets, because of this, right?

Reply:
What do you mean extra?
It creates the temp private key file in the machine folder and creates the public key in the Registry